wix/cups-windows/README.md

```diff
@@ -0,0 +1,33 @@
+# Windows Installer
+
+## Build Requirements
+The WIX toolset is required to build the Windows Installer file, the WIX toolset is required. 
+It can be downloaded from http://wixtoolset.org.
+
+## Build Instructions
+Run candle.exe to build wixobj file from the wxs file:
+candle.exe -arch x64 cups-connector.wxs
```

Comment:
The Windows version of the CUPS Connector is "Windows Connector" or "Cloud Print Windows Connector". Please replace all instances of "cups" or "CUPS" with "windows" or "Windows".